package.json

```diff
@@ -18,7 +18,8 @@
   ],
   "scripts": {
     "build": "ember build",
-    "start": "ember server",
+    "lint:js": "eslint ./*.js addon addon-test-support app config lib server test-support tests",
```

Comment:
this is a new thing for ember-cli addons. it also adds this to travis, but it's failing on storefront for reasons i dont feel are important.